types/gulp-stylelint/gulp-stylelint-tests.ts

```diff
@@ -0,0 +1,19 @@
+import gulp = require('gulp');
+import gulpStylelint = require('gulp-stylelint');
+
+gulp.task('lint-css', function lintCssTask() {
+    return gulp.src('src/**/*.css').pipe(
+        gulpStylelint({
+            failAfterError: true,
+            reportOutputDir: 'reports/lint',
+            reporters: [
+                { formatter: 'verbose', console: true },
+                { formatter: 'json', save: 'report.json' },
+            ],
+            debug: true,
+            fix: true,
+        }),
+    );
+});
+
+gulpStylelint.formatters;
```

Comment:
Is there an expected type/usage for this? Seems a bit odd to leave this here without anything to test that it's correct besides being present.

Reply:
Thanks for your review. it is resolved